applications/GeoMechanicsApplication/custom_conditions/T_normal_flux_condition.hpp

```diff
@@ -0,0 +1,118 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi
+//
+//
+//
+
+
+#if !defined(KRATOS_GEO_T_NORMAL_FLUX_CONDITION_H_INCLUDED )
+#define  KRATOS_GEO_T_NORMAL_FLUX_CONDITION_H_INCLUDED
```

Comment:
now we use the `#pragma once`

Reply:
do you mean I need to remove lines 17 and 18, and replace them by #pragma once ?

Reply:
Done